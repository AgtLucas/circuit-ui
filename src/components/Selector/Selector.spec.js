import React from 'react';

import Selector from '.';

describe('Selector', () => {
  it('should render a default selector appropriately', () => {
    const component = <Selector />;
    expect(component).toMatchSnapshot();
  });
  it('should render a disabled selector appropriately', () => {
    const component = <Selector disabled />;
    expect(component).toMatchSnapshot();
  });
  it('should render a selected selector appropriately', () => {
    const component = <Selector selected />;
    expect(component).toMatchSnapshot();
  });
});
