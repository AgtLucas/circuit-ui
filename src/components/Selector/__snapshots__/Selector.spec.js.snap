// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selector should render a default selector appropriately 1`] = `
<Selector
  disabled={false}
  selected={false}
/>
`;

exports[`Selector should render a disabled selector appropriately 1`] = `
<Selector
  disabled={true}
  selected={false}
/>
`;

exports[`Selector should render a selected selector appropriately 1`] = `
<Selector
  disabled={false}
  selected={true}
/>
`;
